package.json

```diff
@@ -109,10 +108,10 @@
     }
   },
   "engines": {
-    "node": ">= 20"
+    "node": ">=20.11.0"
   },
```

Comment:
We should remove the node engine from the package because in the consuming app it doesn't matter what version of node it's using

Reply:
Should I even leave the "engines" section in there at all then?

Reply:
no we don't need the engines section any more 👍